### DISCUSSION TITLE
Why can't localhost be used as a build server or swarm manager?

### DISCUSSION THREAD

**Discussion by KaelWD** [coollabsio/coolify#3798] [Improvement Requests]
Should this be `!isCloud()` instead?
https://github.com/coollabsio/coolify/blob/1c6d47f2fc1322830574caa20d3f79fb0bcb94b1/resources/views/livewire/server/form.blade.php#L121-L123

**Comment by mike-lloyd03** [comment]
I'm wondering the same thing myself.

**Reply by arcofs** [reply]
+1

**Comment by itsjavi** [comment]
I was wondering the same... for my simple setup I'd rather having a single server to host coolify + build containers + host a docker repository, and then another server to run my site(s).
